File: app/src/main/java/com/study/fldemo/manager/SecondViewI.java
package com.study.fldemo.manager;

import com.study.fldemo.bean.FuLiBean;

import java.util.List;

/**
 * Created by Administrator on 2017/8/8.
 */

public interface SecondViewI {
    void getFuLiListDataSuccess(List<FuLiBean> results);

    void getFuLiListDataFail(String str);
}

File: app/src/main/java/com/study/fldemo/manager/SecondViewI.kt
package com.study.fldemo.manager

import com.study.fldemo.bean.FuLiBean

/**
 * Created by Administrator on 2017/8/8.
 */

interface SecondViewI {
    fun getFuLiListDataSuccess(results: MutableList<FuLiBean>)

    fun getFuLiListDataFail(str: String)
}
